sanity/schemas/components/quote.js

```diff
@@ -19,10 +19,13 @@ export default {
   preview: {
     select: {
       title: 'heading',
+      subtitle: 'quote',
     },
-    prepare({title}) {
+    prepare({title, subtitle}) {
       return {
-        title: `${removeMarkdown(title)}`,
+        title: title
+          ? `[Cytat] - ${removeMarkdown(title)}`
+          : `[Cytat] - ${removeMarkdown(subtitle)}`,
```

Comment:
Isn't it be more clear to write it like: `title: `[Cytat] - ${removeMarkdown(title || subtitle)}``